src/main/java/ru/yandex/clickhouse/response/parser/ClickHouseLocalTimeParser.java

```diff
@@ -0,0 +1,63 @@
+package ru.yandex.clickhouse.response.parser;
+
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.TimeZone;
+
+import ru.yandex.clickhouse.response.ClickHouseColumnInfo;
+
+final class ClickHouseLocalTimeParser extends ClickHouseDateValueParser<LocalTime> {
+
+    private static ClickHouseLocalTimeParser instance;
+
+    static ClickHouseLocalTimeParser getInstance() {
+        if (instance == null) {
+            instance = new ClickHouseLocalTimeParser();
+        }
+        return instance;
+    }
+
+    private ClickHouseLocalTimeParser() {
+        super(LocalTime.class);
+    }
+
+    @Override
+    LocalTime parseDate(String value, ClickHouseColumnInfo columnInfo,
+        TimeZone timeZone)
+    {
+        return LocalTime.MIDNIGHT;
```

Comment:
Are you sure, that it is ok?

Reply:
It is the most straightforward response in my opinion. The ClickHouse column is of type Date, e.g. "2020-03-07". If we adjusted midnight server time zone into JVM time zone, we would perhaps cause more confusion.